observability/opentelemetry/v2/client/otel_options.go

```diff
@@ -0,0 +1,40 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package client
+
+import (
+	"go.opentelemetry.io/otel/attribute"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/observability"
+)
+
+const (
+	// TODO: Should this be the name of this module or the whole package?
+	instrumentationName = "github.com/cloudevents/sdk-go/observability/opentelemetry/v2"
+)
+
+type OTelObservabilityServiceOption func(*OTelObservabilityService)
+
+func WithSpanAttributesGetter(attrGetter func(cloudevents.Event) []attribute.KeyValue) OTelObservabilityServiceOption {
```

Comment:
These could use some docs, e.g., here saying that those are added to the built-in attributes and don't replace them.

Reply:
Done